lib/spoom/sorbet.rb

```diff
@@ -55,7 +55,7 @@ def self.srb_version(*arg, path: '.', capture_err: false)
       out.split(" ")[2]
     end
 
-    sig { params(arg: String, path: String, capture_err: T::Boolean).returns(T.nilable(Metrics)) }
+    sig { params(arg: String, path: String, capture_err: T::Boolean).returns(T.nilable(T::Hash[String, Integer])) }
```

Comment:
`Metrics` was a struct with only one field: the hash.

Let's remove the middleman.